exercises/run-length-encoding/rle_test.exs

```diff
@@ -28,27 +48,27 @@ defmodule RunLengthEncoderTest do
   end
 
   @tag :pending
-  test "encode simple" do
-    assert RunLengthEncoder.encode("AABBBCCCC") == "2A3B4C"
+  test "decode string with no single characters" do
+    assert RunLengthEncoder.decode("2A3B4C") == "AABBBCCCC"
   end
 
   @tag :pending
-  test "decode simple" do
-    assert RunLengthEncoder.decode("2A3B4C") == "AABBBCCCC"
+  test "decode single characters with repeated characters" do
+    assert RunLengthEncoder.decode("12WB12W3B24WB") === "WWWWWWWWWWWWBWWWWWWWWWWWWBBBWWWWWWWWWWWWWWWWWWWWWWWWB"
   end
 
   @tag :pending
-  test "encode with single values" do
-    assert RunLengthEncoder.encode("WWWWWWWWWWWWBWWWWWWWWWWWWBBBWWWWWWWWWWWWWWWWWWWWWWWWB") === "12WB12W3B24WB"
+  test "decode multiple whitespace mixed in string" do
+    assert RunLengthEncoder.decode("2 hs2q q2w2 ") === "  hsqq qww  "
   end
 
   @tag :pending
-  test "decode with single values" do
-    assert RunLengthEncoder.decode("12WB12W3B24WB") === "WWWWWWWWWWWWBWWWWWWWWWWWWBBBWWWWWWWWWWWWWWWWWWWWWWWWB"
+  test "decode lower case string" do
+    assert RunLengthEncoder.decode("2a3b4c") === "aabbbcccc"
   end
 
   @tag :pending
-  test "decode(encode(...)) combination" do
+  test "consistency encode followed by decode gives original string" do
```

Comment:
@parkerl So what are your thoughts on taking the exact descriptions from the `canonical_data.json` file? In general I think it's good to stay consistent, but this one just reads so weird that I think it might be worth changing.